samples/Integration/CMake-fetch-content-app/src/main.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Application that consumes the Azure SDK for C++.
+ *
+ * @remark Set environment variable `STORAGE_CONNECTION_STRING` before running the application.
+ *
+ */
+
+#include <azure/storage/blobs.hpp>
+
+#include <exception>
+#include <iostream>
+
+using namespace Azure::Storage::Blobs;
+
+int main(int argc, char* argv[])
+{
+  (void)argc;
+  (void)argv;
+
+  /**************** Container sdk client ************************/
+  /****************   Create container  ************************/
+  try
+  {
+    auto containerClient = BlobContainerClient::CreateFromConnectionString(
+        std::getenv("STORAGE_CONNECTION_STRING"), "sample");
```

Comment:
```suggestion
    auto connectionString = std::getenv("STORAGE_CONNECTION_STRING");
    if (!connectionString)
    {
      std::cout << "Set the 'STORAGE_CONNECTION_STRING' environment variable." << std::endl;
      return 1;
    }
    auto containerClient = BlobContainerClient::CreateFromConnectionString(
        connectionString, "sample");
```

Reply:
`std::getenv` returns null pointer if the variable is not found.  We can't do what you suggest.
We would need to return to a pointer and then check if not null and do what you say.

However, the focus is not the code that needs to run. The focus is the integration.

Once the integration is done, the customer would want to go and check the package samples and copy paste from there. This samples are not interested in providing guidance for using the service client, but for consuming it

Reply:
> We can't do what you suggest.

Why? Does the` if (!connectionString)` check not work to guard against `null`?